packages/logger/src/logger/ConsoleTransport.ts

```diff
@@ -0,0 +1,30 @@
+// This transport enables Winston logging to the console.
```

Comment:
Was any content of these files modified? Just want to make sure we specifically review anything that changed!

Reply:
No, I've just copy pasted files. None of them have been modified